index.js

```diff
@@ -2015,6 +2015,23 @@
     return new RegExp(source, flags);
   });
 
+  //# regexEscape :: String -> String
+  //.
+  //. Takes a string which may contain regular expression metacharacters,
+  //. and returns a string with those metacharacters escaped.
+  //.
+  //. Properties:
+  //.
+  //.   - ∀ `s` ∈ String: `S.test(S.regex('', S.regexEscape(s)), s)`
```

Comment:
Thanks for the feedback, Scott. I :heart: both `forall s :: String.` and `= true`. I'll make these changes. :)